service-loadbalancer/service_loadbalancer.go

```diff
@@ -277,6 +301,13 @@ func (cfg *loadBalancerConfig) write(services map[string][]service, dryRun bool)
 	conf["startSyslog"] = strconv.FormatBool(cfg.startSyslog)
 	conf["services"] = services
 
+	conf["sslCert"] = ""
+	if cfg.sslCert != "" {
+		conf["sslCert"] = "crt " + cfg.sslCert
+	}
+	if cfg.sslCaCert != "" {
+		conf["sslCert"] = conf["sslCert"].(string) + " ca-file " + cfg.sslCaCert
```

Comment:
Actually, since you're getting it at loadbalancer bootstrap time, it will never change so there's no need to recreate it on each reload. How about just storing it as cfg.sslConfig, and writing a comment with an example of how it might look?